.template-lintrc.js

```diff
@@ -2,4 +2,9 @@
 
 module.exports = {
   extends: 'octane',
+
+  rules: {
+    'no-curly-component-invocation': { allow: ['setTheme'] },
+    'no-implicit-this': { allow: ['setTheme'] },
```

Comment:
Could you please add reference to why was this added?

Reply:
![Screenshot (139)](https://user-images.githubusercontent.com/55057874/140637374-6182570a-30de-42d0-842f-46dd499abe84.png)